### PR TITLE
Added the functionality to show the last 10 opened/used dialogs

### PR DESCRIPTION
**Pull request by deaspo**

### CODE REVIEW

**Comment by dannyparsons** [comment]
Great to see something being done on this. A lot of it looks good and useful. There's a couple of things I see that could improve the code.
It's a bit unnecessary to have a list of all forms and then loop through them to find the one you need. 
We can do this better if your `mnuItems` was a list of Forms instead of a list of Strings. So you are then storing the last 10 dialogs directly in a list of Forms. Then, to get the strings you want for the menu, you can just use .text (or whatever it is) from your dialogs in the list to get a string display. This way you wouldn't also need the `allForms` list to loop through because you have the dialogs directly.

**Comment by deaspo** [comment]
Made changes to convert list of strings to list of forms, removed lists of all forms.

**Comment by deaspo** [comment]
Hi @dannyparsons  I have made changes and removed allForms

File: instat/frmMain.vb [hunk 1]
```diff
@@ -29,11 +29,15 @@ Public Class frmMain
...
     Public dlgLastDialog As Form
+
+    Dim mnuItems As New List(Of Form)
+
     'This is the default data frame to appear in the data frame selector
     'If "" the current worksheet will be used
     'TODO This should be an option in the Options dialog
     '     User can choose a default data frame or set the default as the current worksheet
     Public strDefaultDataFrame As String = ""
+    Dim allForms As New List(Of Form)
```
Comment (dannyparsons) [inline instat/frmMain.vb]: So you can remove this now?

**Comment by deaspo** [comment]
If allowed I can modify this method display the Last 10 opened/used, Edit last dialog and most recently opened items @dannyparsons ?

File: instat/frmMain.vb (hunk 1 above)
Comment (deaspo) [inline instat/frmMain.vb]: Sure I can

**Comment by dannyparsons** [comment]
I think this is looking good now. I can merge it and then we can test it out to see if it needs improvements. And you can still be working on it for a new pull request if you want to keep going.

**Pull request merged**